### PR TITLE
Refactor2402 wip v2

### PR DESCRIPTION
**Pull request by jafariv**
Bon

### CODE REVIEW

File: coupling/services/CouplingCellService.cpph [hunk 1]
```diff
@@ -268,15 +268,16 @@ tarch::la::Vector<dim, double> coupling::services::CouplingCellServiceImpl<Linke
   // initialise with position of global first ghost cell
   //    coupling::indexing::CellIndex<3, coupling::indexing::IndexTrait::vector,
   //    coupling::indexing::IndexTrait::local> localIndex({0, 0, 0});
-  I03 localIndex({0, 0, 0});
+/*  I03 localIndex({0, 0, 0});
```
Comment (rubenhorn) [inline coupling/services/CouplingCellService.cpph:271]: Why is this only commented out? (If the code is not needed, it should be removed, IMHO)

**Review by rubenhorn — COMMENTED** [review]
Only one small comment, otherwise LGTM.

File: coupling/services/CouplingCellService.cpph (hunk 1 above)
Comment (Thinkpiet) [reply coupling/services/CouplingCellService.cpph]: Done!

**Comment by Thinkpiet** [comment]
Nice

**Pull request merged**